crates/ruff_linter/src/rules/pydoclint/snapshots/ruff_linter__rules__pydoclint__tests__docstring-extraneous-parameter_DOC102_google.py.snap

```diff
@@ -0,0 +1,182 @@
+---
+source: crates/ruff_linter/src/rules/pydoclint/mod.rs
+---
+DOC102_google.py:6:1: DOC102 `a` is not in the function's signature
+   |
+ 4 |       Adds two numbers and returns the result.
+ 5 |   
+ 6 | /     Args:
+ 7 | |         a (int): The first number to add.
+ 8 | |         b (int): The second number to add.
+ 9 | | 
+10 | |     Returns:
+   | |_^ DOC102
+11 |           int: The sum of the two numbers.
+12 |       """
+   |
+   = help: Remove the extraneous parameter from the docstring
+
+DOC102_google.py:21:1: DOC102 `multiplier` is not in the function's signature
+   |
+19 |       Multiplies each element in a list by a given multiplier.
+20 |   
+21 | /     Args:
+22 | |         lst (list of int): A list of integers.
+23 | |         multiplier (int): The multiplier for each element in the list.
+24 | | 
+25 | |     Returns:
+   | |_^ DOC102
+26 |           list of int: A new list with each element multiplied.
+27 |       """
+   |
+   = help: Remove the extraneous parameter from the docstring
+
+DOC102_google.py:36:1: DOC102 `numbers` is not in the function's signature
+   |
+34 |       Finds the maximum value in a list of numbers.
+35 |   
+36 | /     Args:
+37 | |         numbers (list of int): A list of integers to search through.
+38 | | 
+39 | |     Returns:
+   | |_^ DOC102
+40 |           int: The maximum value found in the list.
+41 |       """
+   |
+   = help: Remove the extraneous parameter from the docstring
+
+DOC102_google.py:50:1: DOC102 These parameters are not in the function's signature: `name`, `age`, `email`
+   |
+48 |       Creates a user profile with basic information.
+49 |   
+50 | /     Args:
+51 | |         name (str): The name of the user.
+52 | |         age (int): The age of the user.
+53 | |         email (str): The user's email address.
+54 | |         location (str): The location of the user.
+55 | | 
+56 | |     Returns:
+   | |_^ DOC102
+57 |           dict: A dictionary containing the user's profile.
+58 |       """
+   |
+   = help: Remove the extraneous parameters from the docstring
+
+DOC102_google.py:72:1: DOC102 `tax_rate` is not in the function's signature
```

Comment:
Nit: What do you think of "Documented parameter `tax_rate` is not in the function's signature.